build/build.js

```diff
@@ -108,4 +121,12 @@ gulp.task('clean', function () {
     $.del([path.join(conf.paths.dist, '/'), path.join(conf.paths.tmp, '/'), conf.paths.zip]);
 });
 
-gulp.task('build', ['html', 'fonts', 'other']);
+gulp.task('uglify', function (cb) {
+    pump([
+        gulp.src('dist/scripts/*.js'),
```

Comment:
This only uglifies the JS, we can make it like that and do the same with HTML/CSS if that is what you want.